atlp-rwanda/trojans-ec-fe
# #000003  update sales status

## what does this PR do 

this PR is for a seller to update the sale status of the product that is bought in his collection

## Description of the completed task 

as a seller you need to update the sales status of your product so that a buyer can be able to give a review on your product and also in order to be able to view seller stats 

## How can it be tested manually 

to test this feature you need to login in 2 different browser one as a buyer and one as a seller and then you can buy a product in that seller collection and then proceed to the seller's dashboard and to the order you will see the order you have from that buyer and as a seller you can update the status of your sale

## Any background context you want to provide

N/A

## what are the relevant pivotal tacker stories 

N/A

screenShots(if appropriate)

<img width="719" alt="Screenshot 2023-05-12 at 12 44 51 PM" src="https://github.com/atlp-rwanda/trojans-ec-fe/assets/111174174/02f8de1f-e42c-4d46-b7d5-f7672d850d53">

<img width="719" alt="Screenshot 2023-05-12 at 12 45 10 PM" src="https://github.com/atlp-rwanda/trojans-ec-fe/assets/111174174/09253b5a-dd09-408b-aa80-f169cc02d52a">

<img width="719" alt="Screenshot 2023-05-12 at 12 45 42 PM" src="https://github.com/atlp-rwanda/trojans-ec-fe/assets/111174174/429347b2-fcd8-4e65-9aac-c41165fbd2d7">
